docs/extend/extend-apiml/zowe-api-mediation-layer-security-overview.md

```diff
@@ -33,6 +33,18 @@ API ML uses the following authentication methods:
 - **TLS client certificates**
     - Certificates are used for service-only requests
 
+- **OIDC authentication**
+ 
+  API ML is now able to authenticate mainframe users with external/distributed IDP (OIDC provider), such as OKTA, KeyCloak and others.
```

Comment:
```suggestion
  API ML can authenticate mainframe users with external/distributed IDP (OIDC provider), such as OKTA, KeyCloak and others.
```

Reply:
the other document refers to it as IAM system

Reply:
Changed to "can be configured to authenticate ..."